pkg/ccl/backupccl/backup_planning.go

```diff
@@ -1661,13 +1661,32 @@ func getBackupDetailAndManifest(
 		}
 	}
 
+	localityKVs := make([]string, len(urisByLocalityKV))
+	i := 0
+	for k := range urisByLocalityKV {
```

Comment:
the keys in `urisByLocalityKV` only contain the non-default localities, which implies this only checks that the nondefault localities match.

We may already do this somewhere else, but it would be good to double check and explicitly test that we assert the default URI matches throughout the chain (except for incrementals specified with `incremental_location`-- here, incrementals should all have the same defaultURI, but the full backup will have a different one).

Reply:
Resolved by comment below